pkgs/development/libraries/fpattern/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.9";
+  pname = "fpattern";
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    owner = "Loadmaster";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-/QvMQCmoocaXfDm3/c3IAPyfZqR6d7IiJ9UoFKZTpVI=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out $dev/include
+    cp *.c *.h $dev/include
```

Comment:
Their build system should install this.

Reply:
agreed, but it really is [just a handful of .c and .h files](https://github.com/NixOS/nixpkgs/pull/239451). Not even a Makefile.

Reply:
sigh...